code/game/objects/structures/kitchen_spike.dm

```diff
@@ -149,4 +149,84 @@
 	new /obj/item/stack/rods(loc, 4)
 	qdel(src)
 
+/obj/structure/kitchenspike/cross
+	name = "cross"
+	icon = 'icons/obj/cross.dmi'
+	icon_state = "cross"
+	desc = "Degenerates like you belong on one of these."
+	density = 1
+	anchored = 1
+	buckle_lying = 0
+	can_buckle = 1
+	bound_height = 64
+
+/obj/structure/kitchenspike/cross/attackby(obj/item/I, mob/user, params)
```

Comment:
Put this check as a crowbar_act instead of checking crowbar in attackby. Look at the current one in kitchen_spike.dm

Reply:
Ok.